diff --git a/node_modules/react-native-snackbar-component/index.js b/node_modules/react-native-snackbar-component/index.js
index 47fb9a3..5f64d3a 100644
--- a/node_modules/react-native-snackbar-component/index.js
+++ b/node_modules/react-native-snackbar-component/index.js
@@ -113,6 +113,7 @@ class SnackbarComponent extends Component {
         duration: durationValues.entry,
         toValue: 1,
         easing: easingValues.entry,
+        useNativeDriver: false,
       }).start();
       if (nextProps.autoHidingTime) {
         const hideFunc = this.hideSnackbar.bind(this);
@@ -149,6 +150,7 @@ class SnackbarComponent extends Component {
       duration: durationValues.exit,
       toValue: 0,
       easing: easingValues.exit,
+      useNativeDriver: false,
     }).start();
   }
 }
